cmd/api/src/services/fileupload/file_upload.go

```diff
@@ -102,7 +105,17 @@ func WriteAndValidateZip(src io.Reader, dst io.Writer) error {
 
 func WriteAndValidateJSON(src io.Reader, dst io.Writer) error {
 	tr := io.TeeReader(src, dst)
-	_, err := ValidateMetaTag(tr, true)
+	bufReader := bufio.NewReader(tr)
+	if b, err := bufReader.Peek(3); err != nil {
+		return err
+	} else {
+		if b[0] == UTF8BOMBytes[0] && b[1] == UTF8BOMBytes[1] && b[2] == UTF8BOMBytes[2] {
```

Comment:
If you're going to compare against slice indexes like this could I request that you simply assign the BOM values to byte constants?